### PR TITLE
Chore/lw 9182 restore plutus and native script tests

### PR DESCRIPTION
**Pull request by mirceahasegan**
# Context

Cardano Node 8.8.0 upgrade re-enables plutus and native script support.

# Proposed Solution
- Re-enable local-network native/plutus scripts features enabled.
- Re-enable related unit and e2e tests

# Important Changes Introduced
A few tests are still failing, and are being tracked separately.

### CODE REVIEW

File: packages/cardano-services/test/data-mocks/ledger.ts [hunk 1]
```diff
@@ -2,351 +2,22 @@ export const parameters = {
...
-      'verifyEcdsaSecp256k1Signature-memory-arguments': 10,
-      'verifyEd25519Signature-cpu-arguments-intercept': 9_462_713,
-      'verifyEd25519Signature-cpu-arguments-slope': 1021,
-      'verifyEd25519Signature-memory-arguments': 10,
-      'verifySchnorrSecp256k1Signature-cpu-arguments-intercept': 38_887_044,
-      'verifySchnorrSecp256k1Signature-cpu-arguments-slope': 32_947,
-      'verifySchnorrSecp256k1Signature-memory-arguments': 10
-    }
+    PlutusV1: [
+      197_209, 0, 1, 1, 396_231, 621, 0, 1, 150_000, 1000, 0, 1, 150_000, 32, 2_477_736, 29_175, 4, 29_773, 100, 29_773,
```
Comment (mkazlauskas) [inline packages/cardano-services/test/data-mocks/ledger.ts:6]: Doesn't this require a change in mapper? :thinking:

**Review by mkazlauskas — APPROVED** [review]
Nice work! Small commits makes it very easy to review :sunglasses: 

I would classify local-network config changes as `chore` commit type

**Review by AngelCastilloB — APPROVED** [review]
Awesome job @mirceahasegan 🏰

File: packages/cardano-services/test/data-mocks/ledger.ts (hunk 1 above)
Comment (mirceahasegan) [reply packages/cardano-services/test/data-mocks/ledger.ts:6]: No. The mapper takes whatever db-sync has in the costs json, but does not attempt to change the contents.

**Review by iccicci — APPROVED** [review]
Great job!
Excellent granularity with the small commits as well

**Pull request merged**